api/envoy/config/listener/v3/listener.proto

```diff
@@ -147,6 +148,10 @@ message Listener {
   // :ref:`FAQ entry <faq_how_to_setup_sni>`.
   repeated FilterChain filter_chains = 3;
 
+  // Listener FCDS: Listener Filter Chain Discovery Service config block. filter_chains and fcds are mutually exclusive
+  // TBD: filter_chains and fcds_config to be made oneof in future major release
```

Comment:
Please use one-of promotion tags, and "next-major-version" comments (see [style-guide](https://github.com/envoyproxy/envoy/blob/main/api/STYLE.md#field-level)).

Reply:
Fixed